3-mcstas/mcstas-powder-diffraction.ipynb

```diff
@@ -1561,6 +1561,44 @@
    "source": [
     "plot(data_sample_lbco, orders_of_mag=5)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1765475",
+   "metadata": {
+    "tags": [
+     "remove-cell",
+     "dmsc-school-keep"
+    ]
+   },
+   "source": [
+    "## Upload Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0a0328f7",
+   "metadata": {
+    "tags": [
+     "remove-cell",
+     "dmsc-school-keep"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from scicat_helper import upload_widget\n",
```

Comment:
@YooSunYoung Question: should we not also have such cells in the reduction & analysis notebooks to add derived data to Scicat? Or are the students supposed to write them themselves?

Reply:
We will have a data storage exercise on Tuesday morning so we can just use that notebook for data reduction and analysis scicat exercise.

But Mcstas has many parameters for scientific metadata so we thought it would be easy to just make the widget in the same notebook.